app/src/main/java/org/eyeseetea/malariacare/domain/usecase/push/PushUseCase.java

```diff
@@ -44,7 +45,7 @@
         void onApiCallError(ApiCallException e);
     }
 
-    private IPushController mPushController;
+    public IPushController mPushController;
```

Comment:
is dangerous refactor to public pushController for be invokes from strategy but can be invoked from presentation layer also, in PushUseCase only execute method should be public then I think is better refactor for pass pushcontroller as argument in strategy constructor